src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -143,10 +152,27 @@ def _accumulate_load_trades(load, grid, accumulated_trades, is_cell_tower):
     return accumulated_trades
 
 
+def _accumulate_producer_trades(load, grid, accumulated_trades):
+    accumulated_trades[load.name] = {
+        "type": load.name,
```

Comment:
Same goes for this, for uniformity I would suggest to keep the original purpose of type to map to either cell tower or house or unknown. You have the load name from the dict key anyways